### ISSUE TITLE
TypeError: Could not build a TypeSpec for KerasTensor(type_spec=TensorSpec(shape=(None, None, 4), dtype=tf.float32, name=None), name='tf.math.truediv/truediv:0', description="created by layer 'tf.math.truediv'") of unsupported type <class 'keras.engine.keras_tensor.KerasTensor'>.

### ISSUE THREAD

**Issue by Abecadarian** [matterport/Mask_RCNN#2867]
When I try to train on a new set of images I get this error:

TypeError: Could not build a TypeSpec for KerasTensor(type_spec=TensorSpec(shape=(None, None, 4), dtype=tf.float32, name=None), name='tf.math.truediv/truediv:0', description="created by layer 'tf.math.truediv'") of unsupported type <class 'keras.engine.keras_tensor.KerasTensor'>.

**Comment by cnavarrete** [comment]
Which version of keras and tensorflow are you using? I recommend using conda for this knd of stuff since it's likely you are gonna have a hard time porting this repo to tf2

**Comment by San-dow** [comment]
> Which version of keras and tensorflow are you using? I recommend using conda for this knd of stuff since it's likely you are gonna have a hard time porting this repo to tf2

yes,I have to use tf>=2.6 and CUDA>=11.1, due to my gpu RTX30 series :{ . Is there a solution that runs successfully on TF2 for Mask_RCNN-2.1 ？
Appreciate !!!

**Comment by ajayhakkumar** [comment]
Same error
TypeError: Could not build a TypeSpec for KerasTensor(type_spec=TensorSpec(shape=(None, None, 4), dtype=tf.float32, name=None), name='tf.math.truediv/truediv:0', description="created by layer 'tf.math.truediv'") of unsupported type 
<class 'keras.engine.keras_tensor.KerasTensor'>.

**Comment by ElonGong** [comment]
Same error
TypeError: Could not build a TypeSpec for KerasTensor(type_spec=TensorSpec(shape=(None, None, 4), dtype=tf.float32, name=None), name='tf.math.truediv/truediv:0', description="created by layer 'tf.math.truediv'") of unsupported type
<class 'keras.engine.keras_tensor.KerasTensor'>.

**Comment by JennyWow111** [comment]
同问
